src/redux/boards/actions.ts

```diff
@@ -78,7 +78,18 @@ export const actionCreators = {
                 error => dispatch(actionCreators.fetchBoardRequestError(error.error.error))
             )
         }
-    }
+    },
+
+    /*
```

Comment:
Why in comment ?

Reply:
It was what I have after pull, I didn't touch it